fenicsadapter/fenicsadapter.py

```diff
@@ -133,43 +115,17 @@ def update_coupling_expression(self, coupling_expression, data):
         """
         coupling_expression.update_boundary_data(data, self._coupling_mesh_vertices[:, 0], self._coupling_mesh_vertices[:, 1])
 
-    def create_point_sources(self, fixed_boundary, data=None):
+    def create_point_sources(self, fixed_boundary):
         """
-        Create point sources with reference to fixed boundary in a FSI simulation.
+        Set reference to fixed boundary in a FSI simulation needed for point sources.
 
         Parameters
         ----------
         fixed_boundary : Object of class dolfin.fem.bcs.AutoSubDomain
             SubDomain consisting of a fixed boundary condition. For example in FSI cases usually the solid body
             is fixed at one end (fixed end of a flexible beam).
-        data : array_like
-            The coupling data. A numpy array [N x D] where N = number of vertices and D = dimensions of the data, i.e.
-            for scalar valued data D = 1 and for vector valued data D = dimension of problem.
-            If no data is provided then a numpy array of shape (N, D) with zero values is used.
-
-        Returns
-        -------
-        x_forces : list
-            List containing X component of forces with reference to respective point sources on the coupling interface.
-        y_forces : list
-            List containing Y component of forces with reference to respective point sources on the coupling interface.
         """
-        if self._non_standard_initialization and data is None:
-            warn("initialize_data() was called but the data was not supplied for creating the point sources")
-
-        n_vertices, _ = self._coupling_mesh_vertices.shape
-
-        if data is None:
-            # standard initialization with zero valued data
-            if self._read_function_type is FunctionType.SCALAR:
-                data = np.zeros(n_vertices)
-            elif self._read_function_type is FunctionType.VECTOR:
-                data = np.zeros((n_vertices, self._fenics_dimensions))
-            else:
-                raise Exception("No valid read_function is provided in initialization. Cannot create point sources")
-
         self._Dirichlet_Boundary = fixed_boundary
-        return self.update_point_sources(data)
 
     def update_point_sources(self, data):
```

Comment:
If we remove (or rename) `create_point_source`, we should also change the name of this function. For the `CouplingExpression` the pair of `create/update_coupling_expression` makes sense. For the `PointSource` it does not. Suggestion: `get_point_source(self, data)`. The `fixed_boundary` has to be set elsewhere (see above).

Reply:
`get_points_sources(self, data)` accurately describes what this function does :+1: